assets/src/components/reader-theme-selection/index.js

```diff
@@ -56,17 +80,25 @@ export function ReaderThemeSelection( { disableCurrentlyActiveTheme = false } )
 					__( 'Select the theme template for mobile visitors', 'amp' )
 				}
 			</p>
+			{ activeTheme && hideCurrentlyActiveTheme && (
+				<AMPNotice>
+					{
+						sprintf(
+							// Translators: placeholder is the name of a WordPress theme.
```

Comment:
Maybe this doesn't matter, but I've always seen the translator comments formatted as such:

```suggestion
							/* translators: placeholder is the name of a WordPress theme. */
```